tests/market/test_future.py

```diff
@@ -194,7 +184,8 @@ def test_accept_offer(future_market):
         """Test if trade is added to trade buffers when accept_offer is called."""
         first_future_market = next(iter(future_market.slot_bid_mapping))
         offer = future_market.offer(1, 1, "seller", "seller_origin", time_slot=first_future_market)
-        trade = future_market.accept_offer(offer, "buyer")
+        trade = future_market.accept_offer(offer, "buyer",
+                                           trade_bid_info=TradeBidOfferInfo(1, 1, 1, 1, 1))
```

Comment:
Why did you pass the `trade_bid_info` ?

Reply:
I can not remember. I think the test was failing.I just checked, it is indeed not needed.